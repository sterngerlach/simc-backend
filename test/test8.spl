
{
    int i = 2;
    int j = 3;

    int func(int x, int y)
    {
        return 42;
    }

    void hoge(int x, int y)
    {
        int z = x + y;

        int fuga()
        {
            return 2;
        }

        return z;
    }

    hoge(i, j);
}

