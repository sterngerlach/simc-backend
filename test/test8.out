result type mismatch in hoge()
